docs/using/emr.md

```diff
@@ -7,4 +7,36 @@ nav_order: 6
 has_children: false
 ---
 
-# Using lakeFS with EMR
\ No newline at end of file
+# Using lakeFS with EMR
+
+EMR is AWS big data platform for processing data using open source tools such as Spark, Hive and more.
```

Comment:
Please use the description from here (or possibly link to it): https://docs.aws.amazon.com/emr/latest/ManagementGuide/emr-what-is-emr.html